data/scripts/creaturescripts/others/remove_empty_parcel.lua

```diff
@@ -1,18 +1,25 @@
 local removeEmptyParcelsEvent = CreatureEvent("RemoveEmptyParcelsOnLogin")
 
 function removeEmptyParcelsEvent.onLogin(player)
-	local emptyParcelsToRemove = {}
-	for _, parcel in ipairs(player:getStoreInbox():getItems(true)) do
-		if parcel:getId() == ITEM_PARCEL_STAMPED and parcel:getEmptySlots() == 10 then
-			table.insert(emptyParcelsToRemove, parcel)
+	local inbox = player:getStoreInbox()
+	if not inbox then
```

Comment:
Can it happen that the player does not have a store inbox?

Reply:
apparently yes, as this is a standard check in many scripts, and an error was reported on discord.